### DISCUSSION TITLE
Unpark all Local

### DISCUSSION THREAD

**Discussion by ChuckStarchaser** [endless-sky/endless-sky#7532] [Ideas]
The button **[Un-park All]** un-parks any ships you have parked in any system.
I suggest an **[Un-park All Here]** or **[Un-park All Local]** that avoids un-parking ships far away.

Secondly, I have a question:  Is there any use in un-parking ships remotely?
If not, perhaps **[Un-park All]** should simply apply to all local ships parked.

**Comment by Zitchas** [comment]
Unparking ships remotely is a convenient way to have a fleet arrive wherever you are: as soon as you take off, they start making their way to you.

I'm not sure this is a good mechanic to have, since it visibly breaks the whole "one jump is a day" lore. But that is the way things are right now.

**Comment by ChuckStarchaser** [comment]
Ahhh, now I understand!  Thanks.

**Comment by MasterOfGrey** [comment]
Having a local and an all button would be good though - and having them both be further away from the existing buttons would be even better.
(I accidentally unparked 100 ships across a half dozen systems the other day. Was a pain to go and re-park them all when they were ordered in ways unrelated to their status being parked.)

**Comment by ChuckStarchaser** [comment]
I started an issue on this issue, and in my last post on it I suggested

- Renaming [Un-park All]  to [Summon All My Ships], and then
- Creating a new button named [Un-park All] that un-parks all LOCAL ships

**Comment by TomGoodIdea** [comment]
#7549
